__tests__/app.test.js

```diff
@@ -280,3 +280,69 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("PATCH /api/articles/:article_id", () => {
+  it("200: updates the article and responds with the updated article", () => {
```

Comment:
This test description can be a little clearer. What exactly are we updating on article?